backend/migrations.sh

```diff
@@ -10,36 +10,11 @@ else
     echo 'Environment variables set...'
```

Comment:
This might have an impact on the local deployment for devs as the compose uses Dockerfile.local -> local_start.sh -> initDB.sh. We could maybe retain the original initDB.sh and add the migrations.sh.

Reply:
good catch @nikhila-aot , I hope this commit solves the problem, https://github.com/bcgov/nr-site-registry/pull/167/commits/093c65c10a3d5d49d975e5380cc6f5c79c72df4f

the extensions are created as part of typeorm migrations itself, so we dont have to explicitly create them.

let me know if this is good to be resolved now.